src/components/Toast/style.css

```diff
@@ -0,0 +1,265 @@
+.snackbar {
+  animation: slideBottomLeft 300ms forwards,
+    slideoutBottomLeft 300ms forwards 5s;
+  font-family: 'Roboto', sans-serif;
```

Comment:
have been noticing from the top, almost all your components are using Roboto so why not declare it commonly in the body or via * selector, and specify only where a different one is being used.